src/components/Filters.js

```diff
@@ -0,0 +1,127 @@
+export const Filters = () => {
+  return (
+    <aside className="filter-section p-4">
+      <div className="filter-header">
+        <p className="fs-3">Filter</p>
+        <button className="mr-auto btn text-light">Clear filters</button>
+      </div>
+      <div className="my-1">
+        <p className="fs-2">Sort by</p>
+        <select className="input">
+          <option value="" defaultValue>
+            sort by
+          </option>
+          <option value="price_asc">Price: low to high</option>
+          <option value="price_desc">Price: high to low</option>
+          <option value="rating_asc">Rating: low to high</option>
+          <option value="rating_desc">Rating: high to low</option>
```

Comment:
filter on rating should be added under separate select input

Reply:
filters functionality is added separately. This is for sorting based on review